pubtools/_ami/rhsm.py

```diff
@@ -61,8 +61,18 @@ def _on_failure(self, exception):
     def _get(self, *args, **kwargs):
         return self._session.get(*args, **kwargs)
 
-    def _send(self, *args, **kwargs):
-        return self._session.send(*args, **kwargs)
+    def _send(self, prepped_req, **kwargs):
+        settings = {
+            "url": prepped_req.url,
+            "proxies": kwargs.get("proxies") or None,
+            "stream": kwargs.get("stream") or None,
+            "verify": kwargs.get("verify") or None,
+            "cert": kwargs.get("cert") or None,
```

Comment:
Not entirely sure if the `or None` part is a good idea. `kwargs.get()` already returns `None` if the item is not found, so that should cover it. But if either of the items has falsey value (e.g. everyone's favorite `verify=False`), then the `or None` part sets the value to `None` which is then passed to `merge_environment_settings()` and which has a different meaning.